tests/app_regular_xcommon_build/Makefile

```diff
@@ -37,5 +37,5 @@ USED_MODULES = lib_mic_array
 # The following part of the Makefile includes the common build infrastructure
 # for compiling XMOS applications. You should not need to edit below here.
 
-XMOS_MAKE_PATH ?= ../..
+XMOS_MAKE_PATH ?= ../../..
```

Comment:
Shouldn't be necessary; this is an environment variable, set by XTC SetEnv, so this line of the Makefile should have no effect.

Reply:
I have removed them from all xcommon Makefiles